attestation-container/attestation-container.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"os"
+
+	"microsoft/attestation-container/attest"
+	pb "microsoft/attestation-container/protobuf"
+
+	"google.golang.org/grpc"
+)
+
+var (
+	port = flag.Int("port", 50051, "The server port")
+)
+
+type server struct {
+	pb.UnimplementedAttestationContainerServer
+}
+
+func (s *server) FetchAttestation(ctx context.Context, in *pb.FetchAttestationRequest) (*pb.FetchAttestationReply, error) {
+	reportData := [attest.REPORT_DATA_SIZE]byte{}
+	if len(in.GetReportData()) > attest.REPORT_DATA_SIZE {
+		return nil, fmt.Errorf("`report_data` needs to be smaller than %d bytes. size: %d bytes", attest.REPORT_DATA_SIZE, len(in.GetReportData()))
+	}
+	copy(reportData[:], in.GetReportData())
+	reportBytes, err := attest.FetchAttestationReportByte(reportData)
+	if err != nil {
+		return nil, fmt.Errorf("failed to fetch attestation report: %s", err)
+	}
+	return &pb.FetchAttestationReply{Attestation: reportBytes}, nil
+}
+
+func main() {
+	fmt.Println("Attestation container started.")
```

Comment:
Perhaps change this to `log.Println` as that seems to include timestamp as well?